test/confit-test.js

```diff
@@ -26,6 +26,10 @@ test('confit', function (t) {
 
 
     t.test('get', function (t) {
+        //setting process.env.env to development, should not change 'env:env'.
+        //This should be ignored and 'env:env' should solely depend on process.env.NODE_ENV
+        process.env.env = 'development';
```

Comment:
That would not be the case. If you comment out the changes i made here- https://github.com/krakenjs/confit/pull/57/files#diff-1c25f2af06a7bfd6c36abff0f88b9a2bR52. The unit test would fail.